src/Couchbase.Lite.Shared/API/Document/Document.cs

```diff
@@ -155,6 +163,29 @@ public string? RevisionID
             }
         }
 
+        /// <summary>
+        /// The hybrid logical timestamp that the revision was created.
+        /// </summary>
+        public DateTimeOffset? Timestamp
+        {
+            get {
+                using var scope = ThreadSafety.BeginLockedScope();
+                var rawVal = c4Doc?.HasValue == true ? NativeRaw.c4rev_getTimestamp(c4Doc.RawDoc->selectedRev.revID) : 0;
+                if(rawVal == 0) {
+                    return null;
+                }
+
+                // .NET ticks are in 100 nanosecond intervals
+                rawVal /= 100;
```

Comment:
I thought we were just going to return the nanos.
I am returning nanos with a comment that says that that's what they are...